### PR TITLE
#2557 - Remove course load calculation from PT

### PR DESCRIPTION
**Pull request by guru-aot**
- [x] Remove calculatedDataCourseLoadPercentage from calculation in parttime-assessment-*.bpmn (Screenshot)
- [x] Under "Calculate Course Load Tuition Totals": Update calculatedDataActualTuitionCosts = offeringActualTuitionCosts
- [x] Under "Calculate Course Load Tuition Totals": Update calculatedDataMandatoryFees = offeringMandatoryFees
- [x] Under "Calculate Course Load Tuition Totals": Update calculatedDataProgramRelatedCosts = offeringProgramRelatedCosts
- [ ] Under "Calculate Course Load Tuition Totals": Update calculatedDataExceptionalExpenses = offeringExceptionalExpenses
- [ ] Update any affected E2E tests

Update calculatedDataExceptionalExpenses = offeringExceptionalExpenses - is not done as calculatedDataExceptionalExpenses is removed as a part of #2229.
Update any affected E2E tests - No test cases are affected, so no change.

### CODE REVIEW

File: sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2021-2022.bpmn
Comment (ann-aot) [inline sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2021-2022.bpmn:1]: @guru-aot, to double check, so, as part of PT calculation, we don't need ` calculatedDataExceptionalExpenses = offeringExceptionalExpenses`?

File: sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2022-2023.bpmn
Comment (ann-aot) [inline sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2022-2023.bpmn:1]: maybe we can update the modeler, I believe @andrepestana-aot  change is having 5.18

**Review by ann-aot — COMMENTED** [review]
LGTM @guru-aot  👍 Just a question and a suggestion

File: sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2022-2023.bpmn
Comment (guru-aot) [reply sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2022-2023.bpmn:1]: I updated mine to latest, i can check again

File: sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2021-2022.bpmn
Comment (guru-aot) [reply sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2021-2022.bpmn:1]: Yes @ann-aot its already removed

File: sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2022-2023.bpmn
Comment (ann-aot) [reply sources/packages/backend/workflow/BPMN/camunda-8/parttime-assessment-2022-2023.bpmn:1]: yup, you are using the lastest. resolving the comment

**Review by ann-aot — APPROVED** [review]
Thanks for the clarification @guru-aot . Nice work 👍

**Review by lewischen-aot — APPROVED** [review]
LGTM. I'm still new to this calculation process. Please see comments from other devs.

**Review by andrepestana-aot — COMMENTED** [review]
Looks good!

**Pull request merged**